ethereum/contracts/interfaces/relayer/IWormholeReceiver.sol

```diff
@@ -1,48 +1,67 @@
 // SPDX-License-Identifier: Apache 2
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.19;
```

Comment:
Quick question: does that mean that users who want to use this interface file would need to have solidity 0.8.19?